data/LSIR/projects.yaml

```diff
@@ -67,75 +77,85 @@ projects:
     information:
       - type: Demo
         title: Online Technology Demos
-        url: https://modemos.epfl.ch/
-      - type: Demo
-        title: Online Image Retrieval Demo
-        url: https://modemos.epfl.ch/article
```

Comment:
If I'm not mistaken, this demo is not linked from the "main" page

Reply:
You're right, it's not; I'm putting it back.